package.json

```diff
@@ -25,38 +25,39 @@
     "test:tofile": "npm run compile && TZ='GMT' mocha --no-colors -R spec \"test/**/*.js\" 2>&1 | tee test-report.log"
   },
   "devDependencies": {
-    "@babel/cli": "^7.11.5",
-    "@babel/core": "^7.11.5",
-    "@babel/plugin-proposal-json-strings": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.11.5",
-    "@babel/plugin-transform-strict-mode": "^7.0.0",
-    "@babel/preset-env": "^7.11.5",
-    "@babel/runtime": "^7.12.5",
-    "babel-plugin-add-module-exports": "^1.0.2",
+    "@babel/cli": "^7.17.0",
+    "@babel/core": "^7.17.0",
+    "@babel/plugin-proposal-json-strings": "^7.16.7",
+    "@babel/plugin-transform-runtime": "^7.17.0",
+    "@babel/plugin-transform-strict-mode": "^7.16.7",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/runtime": "^7.17.0",
+    "babel-plugin-add-module-exports": "^1.0.4",
     "browserslist-config-nebenan": "^1.0.0",
-    "chai": "^4.2.0",
-    "eslint": "^7.13.0",
-    "eslint-config-nebenan-base": "^1.3.1",
+    "chai": "^4.3.6",
+    "eslint": "^8.8.0",
+    "eslint-config-nebenan-base": "^2.0.1",
     "local-storage-adapter": "^2.0.0",
-    "mocha": "^8.2.1",
+    "mocha": "^9.2.0",
     "proxyquire": "^2.1.3",
-    "react": "^17.0.1",
-    "react-redux": "^7.2.1",
-    "redux": "^4.0.5",
-    "sinon": "^10.0.0"
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-redux": "^7.2.6",
+    "redux": "^4.1.2",
+    "sinon": "^13.0.1"
   },
   "peerDependencies": {
-    "@babel/runtime": "^7.12.5",
+    "@babel/runtime": "^7.17.0",
     "local-storage-adapter": "^2.0.0",
     "react": "^16.14.0 || ^17.0.1",
-    "react-redux": "^7.2.1"
+    "react-redux": "^7.2.6"
```

Comment:
Don't update peerDeps - otherwise you cause breaking changes
https://wiki.nebenan.de/display/AWSM/How+to+work+with+external+packages#Howtoworkwithexternalpackages-Situation1:UpgradingpeerDependencies

Reply:
Still valid @iamuchejude - you forgot to reset babel/runtime

Reply:
Updated.